### PR TITLE
Add usage and license sections in README

### PR DESCRIPTION
**Pull request by b0oh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,43 @@ by adding `secrex` to your list of dependencies in `mix.exs`:
...
+* `key_file` is a path for a file that contain a key that will be used for encrypting and decrypting
+  if attribute is not provided then key will be asked to enter
+* `files` is a list of files that needs to be encrypted and decrypted
+
+### Mix tasks
+
+* `mix secrex.encrypt`
+* `mix secrex.decrypt`
+
+### Useful functions
```
Comment (qcam) [inline README.md]: Helper functions maybe?

File: README.md [hunk 2]
```diff
@@ -10,7 +10,43 @@ by adding `secrex` to your list of dependencies in `mix.exs`:
...
+
+Secrex requires some configuration in order to work. For example, in config/config.exs:
+
+```elixir
+config :secrex,
+  key_file: ".secrets_key",
+  files: ["config/env/prod.secret.exs"]
+```
+
+* `key_file` is a path for a file that contain a key that will be used for encrypting and decrypting
```
Comment (qcam) [inline README.md]: a path _to_ the key file that will be used for encryption and decryption?

Comment (qcam) [reply README.md]: perhaps _if this is not configured, you will be prompted to enter later_?

File: README.md [hunk 3]
```diff
@@ -10,7 +10,43 @@ by adding `secrex` to your list of dependencies in `mix.exs`:
...
+* `mix secrex.encrypt`
+* `mix secrex.decrypt`
+
+### Useful functions
+
+For example if we have a `deploy` task, we can prevent deploy if secrets were diverged
+
+```elixir
+if Mix.Secrex.secret_files_changed?() do
+  Mix.raise(
```
Comment (lexmag) [inline README.md:43]: https://github.com/lexmag/elixir-style-guide/blob/master/README.md#exception-message

File: README.md (hunk 1 above)
Comment (b0oh) [reply README.md]: dun

File: README.md (hunk 2 above)
Comment (b0oh) [reply README.md]: dun

File: README.md (hunk 3 above)
Comment (b0oh) [reply README.md:43]: dun

**Pull request merged**